### PR TITLE
Changes incorporating Review 1

### PR DESCRIPTION
**Pull request by niranjank2022**
- **Update .env and index.js for env variables**
- **Update Readme**
- **Added routes/auth section from admin.js, user.js**
- **Update user side code**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,29 +1,38 @@
...
-app.get("/", (req, res) => {
-    res.send("Hello World!");
-});
+// Adding the middlewares to the app
+app.use(express.json());
 
+// Adding the routes to the app
+app.use(authRoutes);
 app.use("/admin", adminRoutes);
 app.use("/user", userRoutes);
```
Comment (srikoushik) [inline index.js]: Sample

**Review by srikoushik — CHANGES_REQUESTED** [review]

File: index.js [hunk 2]
```diff
@@ -1,29 +1,38 @@
...
 
-const app = express();
-app.listen(3000, () => {
-    console.log("Server is running on port 3000.");
-});
-
-
-app.use(express.json());
+// Import .env variables
+dotenv.config({path: "./.env"});
```
Comment (Rithikaram29) [inline index.js:11]: Omit the path if your .env file is in the root directory

File: index.js [hunk 3]
```diff
@@ -1,29 +1,38 @@
...
 
-app.get("/", (req, res) => {
-    res.send("Hello World!");
-});
+// Adding the middlewares to the app
+app.use(express.json());
 
+// Adding the routes to the app
+app.use(authRoutes);
 app.use("/admin", adminRoutes);
```
Comment (Rithikaram29) [inline index.js]: For admin and user routes, add authorisation middleware for validating admin and user accessing their accounts.

File: index.js (hunk 2 above)
Comment (niranjank2022) [reply index.js:11]: if I comment this line, the env variables are not imported. PORT becomes undefined.

**Review by Rithikaram29 — APPROVED** [review]

**Pull request merged**